README.md

```diff
@@ -1,67 +1,89 @@
 # erouska-android
 
-eRouška ("rouška" means "face mask" in Czech) is an app which helps to fight against COVID-19. When turned on, it scans other users of the app via Bluetooth and saves this data locally. When some user tests positive for COVID-19, he/she is asked to upload their data. Then healtcare authorities can see phone numbers of users of the app which were in close contact with the infected and can call them about further measures.
+## [erouska.cz](https://erouska.cz)
+## [Download from Google Play](https://play.google.com/store/apps/details?id=cz.covid19cz.erouska)
 
-Read more at [our website](https://erouska.cz).
+Read our [**FAQ** (Czech only)](https://erouska.cz/caste-dotazy)
 
-[Download from Google Play](https://play.google.com/store/apps/details?id=cz.covid19cz.erouska)
+eRouška (_rouška_ = _face mask_ in Czech) helps to fight against COVID-19.
 
-## How do we protect user's privacy?
-- All data is saved only locally, it is uploaded only with user's consent
-- User can always turn the scanning off
-- We change broadcasted ID every hour. Nobody can track you using this ID (only our backend has knowledge which ID corresponds to which phone number)
-- We delete data on backend after 6 hours from upload
-- You can always delete your data, including your phone number
-- This app is open-source from the start, so anyone can check our claims are true
+eRouška uses Bluetooth to scan the area around the device for other eRouška users and saves the data of these encounters.
 
-Read more at [our FAQ](https://erouska.cz/caste-dotazy)
+When an eRouška user tests positive for COVID-19, the user is contacted by a healthcare authority and is asked to upload the data to create a map of potential secondary infections.
+
+The healthcare authorities then analyse the data and contact the possibly newly infected for further measures (quarantine, testing).
+
+The App is registered with a phone number. The phone numbers are available only for the healthcare authorities.
+
+
+## How do we protect the user's privacy?
+
+- User can remove all collected data, including the phone number.
+- All data are saved locally on the user's device. Data are uploaded only with the user's consent after a healthcare authority's request.
+- The scanning can be turned off manually at any time.
+- The broadcasted _Device ID_ is changed every hour, so a user cannot be tracked with it. (Only our backend has a knowledge of which _Device ID_'s correspond to which phone number.)
+- The data are kept on backend for 6 hours, then deleted.
+- eRouška is developed open-source from day one.
+
+## Who is developing eRouška?
+
+We are an initiative consisting of people from various Czech IT companies and volunteers called [COVID19CZ](https://covid19cz.cz). We are all unpaid volunteers. 
+
+We are on Slack! [covid19cz.slack.com](covid19cz.slack.com), channel _#erouska_.
```

Comment:
We don't want everyone to join Slack, only the biggest contributors. I would remove this and  only explained it in Contributing section.